roles/elasticsearch/defaults/main.yml

```
@@ -25,7 +25,11 @@ elasticsearch_jna_workaround: false
 elastic_ca_dir: /opt/es-ca
 elastic_initial_passwords: /usr/share/elasticsearch/initial_passwords
 elastic_ca_pass: PleaseChangeMe
-elastic_cert_pass: PleaseChangeMeIndividually
+elasticsearch_tls_key_passphrase: PleaseChangeMeIndividually
+ca_expiration_buffer: 30
```

```suggestion
elastic_ca_expiration_buffer: 30
```

Please keep the prefix. Otherwise you might have collisions with our `ca` role

---

Done.